### PR TITLE
IGNITE-20714 Sql. Test Framework. Support DDL scripts to initialise schema

### PR DESCRIPTION
**Pull request by korlov42**
Thank you for submitting the pull request.

To streamline the review process of the patch and ensure better code quality
we ask both an author and a reviewer to verify the following:

### The Review Checklist
- [ ] **Formal criteria:** TC status, codestyle, mandatory documentation. Also make sure to complete the following:  
\- There is a single JIRA ticket related to the pull request.  
\- The web-link to the pull request is attached to the JIRA ticket.  
\- The JIRA ticket has the Patch Available state.  
\- The description of the JIRA ticket explains WHAT was made, WHY and HOW.  
\- The pull request title is treated as the final commit message. The following pattern must be used: IGNITE-XXXX Change summary where XXXX - number of JIRA issue.
- [ ] **Design:** new code conforms with the design principles of the components it is added to.
- [ ] **Patch quality:** patch cannot be split into smaller pieces, its size must be reasonable.
- [ ] **Code quality:** code is clean and readable, necessary developer documentation is added if needed.
- [ ] **Tests code quality:** test set covers positive/negative scenarios, happy/edge cases. Tests are effective in terms of execution time and resources.

### Notes
- [Apache Ignite Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Java+Code+Style+Guide)

### CODE REVIEW

File: modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java [hunk 1]
```diff
@@ -318,6 +319,10 @@ public CompletableFuture<Void> execute(CatalogCommand command) {
 
     @Override
     public CompletableFuture<Void> execute(List<CatalogCommand> commands) throws IllegalArgumentException {
```
Comment (ygerzhedovich) [inline modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java:321]: seems IllegalArgumentException can be thrown here, let's remove it

File: modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java [hunk 2]
```diff
@@ -515,18 +761,27 @@ public ClusterBuilder end() {
...
-            TableDescriptorImpl tableDescriptor = new TableDescriptorImpl(columns, distribution);
+        private List<CatalogCommand> build() {
+            List<CatalogCommand> commands = new ArrayList<>(1 + indexBuilders.size());
 
-            List<IgniteIndex> indexes = indexBuilders.stream()
-                    .map(idx -> idx.build(tableDescriptor))
-                    .collect(Collectors.toList());
+            commands.add(
+                    CreateTableCommand.builder()
+                            .schemaName("PUBLIC")
```
Comment (lowka) [inline modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java:769]: Perhaps we can expose schema name via builder and assign default value to `PUBLIC` (Catalog::DEFAULT_SCHEMA_NAME would be better). What do you think?

File: modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/benchmarks/TpchHelper.java [hunk 3]
```diff
@@ -65,8 +65,16 @@ public static String getQuery(String queryId) {
         }
     }
 
+    /**
+     * Returns a string representing DDL script with all the tables' and indexes' definitions
```
Comment (ygerzhedovich) [inline modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/benchmarks/TpchHelper.java:69]: what does mean symbols ' in the sentence?

Comment (AMashenkov) [reply modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/benchmarks/TpchHelper.java:69]: "An apostrophe followed by an “s” is the most common way to indicate possession (ownership) in English"

**Review by lowka — APPROVED** [review]

File: modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java (hunk 2 above)
Comment (korlov42) [reply modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java:769]: honestly, I see no point in using different schemas... It's just a namespace that is used during name resolution only

File: modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java (hunk 1 above)
Comment (korlov42) [reply modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java]: removed

**Review by ygerzhedovich — APPROVED** [review]

**Pull request merged**